### ISSUE TITLE
Hi..In my case the generated images are found to be poorer in quality (esp. local structure) unlike SINGAN

### ISSUE THREAD

**Issue by JJashim** [tohinz/ConSinGAN#24]
I have performed image generation using SINGAN with default params set. The generated image seems better in comparison with ConSinGan though i tried tweaking lr_scales =(0.1/0.5/1.0),train_stages = (5/8/10) in consingan but i couldn't generate better samples. Can you please help me understand what tweaks i need to perform (in priority order as i lack a good GPU). 
I prefer to work with ConSinGAN due to its speed & memory. 

Training Image: circuit board. 

-Thanks

**Comment by tohinz** [comment]
Hi, there are some images for which SinGAN may perform better than ConSinGAN. ConSinGAN tends to perform better for images with more structure in them.
In general, the easiest thing to try would be to add batch normalization to the training process (simply add `--batch_norm`). This will slow things down a little but may improve the performance and also fix your other issue #23.
